_data/live_deployments.json

```diff
@@ -108,6 +108,19 @@
         }
       ]
     },
+    {
+      "name": "Galaxy Training Network",
```

Comment:
Can you clarify here in the comment thread the difference between Galaxy Training Network and Galaxy Project? Is it because you have your own sitemap that you've created it as a separate resource?

Reply:
Sure, absolutely. Conceptually the galaxy training network (GTN) is a bit of a "sub team" of Galaxy project, focused on training. Galaxy project itself is primarily focused on the development of Galaxy.